requirements.txt

```diff
@@ -1,6 +1,6 @@
 mkdocs==1.5.2
 markdown==3.3.7 # Temporarily pin version <3.4, see https://github.com/mkdocs/mkdocs/pull/2893
```

Comment:
Let's try this:

```suggestion
markdown==3.4.4
```

Reply:
Great! I think it's worth following up with an investigation, see [DOCS-599](https://codacy.atlassian.net/browse/DOCS-599).